### PR TITLE
Adds support for Byzantium opcodes (WIP)

### LINKED ISSUE #163: Support Byzantium opcodes
- [ ] revert
- [ ] returndatasize
- [ ] returndatacopy
- [ ] staticcall

### PR DESCRIPTION
**Pull request by lrettig**
Creating a PR to track this work in progress. Closes #163.

### CODE REVIEW

File: wasm/generateInterface.js [hunk 1]
```diff
@@ -137,6 +143,18 @@ const interfaceManifest = {
...
   },
+  RETURNDATACOPY: {
+    name: 'returnDataCopy',
+    async: true,
+    input: ['writeOffset', 'i32', 'length'],
+    output: []
+  },
+  RETURNDATASIZE: {
+    name: 'getReturnDataSize',
+    async: true,
```
Comment (axic) [inline wasm/generateInterface.js]: These two are not `async`.

File: opcodes.js [hunk 2]
```diff
@@ -147,6 +149,8 @@ const codes = {
   0xf2: ['CALLCODE', 0, 7, 1],
   0xf3: ['RETURN', 0, 2, 0],
   0xf4: ['DELEGATECALL', 0, 6, 1],
+  0xfa: ['STATICCALL', 0, 6, 1],
+  0xfd: ['REVERT', 0, 2, 0],
```
Comment (axic) [inline opcodes.js]: Should also explicitly handle `0xfe` (INVALID), for clarity.

**Comment by axic** [comment]
Even though this is not finished (not testing the additions), it is still better to merge them as the rest of evm2wasm is broken anyway.

**Comment by jalextowle** [comment]
I'm going to start working on this, if that's alright. I talked to @lrettig about it in the Gitter.

**Comment by lrettig** [comment]
It's been merged. Let me know if you find anything missing; and as @axic indicated there are no tests for the new stuff, so that may be a good place to start.

**Pull request merged**